packages/backend-for-frontend/src/services/validators.ts

```diff
@@ -208,3 +210,13 @@ export function assertIsDraftEservice(eservice: catalogApi.EService): void {
     throw eserviceIsNotDraft(eservice.id);
   }
 }
+
+export function assertEServiceIsTemplateInstance(
```

Comment:
I think this should be called the opposite: we want to assert that the e-service is **NOT** a template instance, and throw if it is (the operation is blocked in case it is a template instance)

```suggestion
export function assertEServiceNotTemplateInstance(
```

Reply:
I started from the existing assertion, but I wasn’t satisfied with the fact that, if I asserted that the eService was **not** an instance of a template, I then had to throw an exception—essentially the opposite of what I actually needed to do.  

However, if the goal is to maintain a single assertion of this kind, that’s fine—I’ll adjust the logic accordingly.